warehouse/models/mart/gtfs_quality/_mart_gtfs_quality.yml

```diff
@@ -0,0 +1,85 @@
+version: 2
+
+models:
+  # TODO: once we have checks that occur above the feed level (e.g. organization)
+  #   we will need to create additional daily check fact tables at the
+  #   appropriate grain
+  - name: fct_daily_feed_guideline_checks
```

Comment:
How would an RT feed check fit into here?

Reply:
RT could fit in but wouldn't have a feed_key right now, since we really only have a schedule feed dimension. Once we produce the quartet concept, rows should also key to that.

Reply:
So, as we discussed, this framework kind of only applies to schedule feeds right now, but it should give enough to start implementing schedule feed checks in a way that will be directly applicable even as we make the quartet/dataset logic more robust (i.e., migrations towards this intermediate state are not wasted work). I am aiming to get my ITP ID mapping doc out today or tomorrow and that would move us towards the quartet concept.

Reply:
Ok thank you, and sorry for making you repeat yourselves!